src/integration_tests/telemetry_async.cpp

```diff
@@ -215,6 +223,20 @@ void print_ground_speed_ned(Telemetry::GroundSpeedNED ground_speed_ned)
     _received_ground_speed = true;
 }
 
+void print_imu_reading_ned(Telemetry::IMUReadingNED imu_reading_ned)
+{
+    std::cout << "Acceleration north: " << imu_reading_ned.acceleration.north_m_s2 << " m/s^2, "
+              << "east: " << imu_reading_ned.acceleration.east_m_s2 << " m/s^2, "
+              << "down: " << imu_reading_ned.acceleration.down_m_s2 << " m/s^2" << std::endl;
+    << "Angular velocity north: " << imu_reading_ned.angular_velocity.north_rad_s << " rad/s, "
+    << "east: " << imu_reading_ned.angular_velocity.east_rad_s << " rad/s, "
+    << "down: " << imu_reading_ned.angular_velocity.down_rad_s << " rad/s" << std::endl;
+    << "Magnetic field north: " << imu_reading_ned.magnetic_field.north_gauss << " G, "
+    << "east: " << imu_reading_ned.magnetic_field.east_gauss << " G, "
+    << "down: " << imu_reading_ned.magnetic_field.down_gauss << " G" << std::endl;
```

Comment:
```suggestion
    << "down: " << imu_reading_ned.magnetic_field.down_gauss << " G, "
```

Reply:
Thanks for the review. I have added the suggestions(for some reason i couldn't commit your suggestions directly so i made another commit to add the desired changes)! The telemetry_async.cpp test still doesn't seem to print the imureading, though.